### DISCUSSION TITLE
Tablet auxiliary buttons sometimes not working

### DISCUSSION THREAD

**Discussion by nanashi-1** [ppy/osu#20660] [Ideas] [answered]
I found this comment on YouTube:
![Screenshot from 2022-10-09 19-17-37](https://user-images.githubusercontent.com/54083498/194753928-e6494a84-b938-4720-a274-e91dd39b9261.png)

Buttons are not working for him/her. Though I think it could be fixed if he/she uses OpenTabletDriver, but would it be better to have this setting in-game?

**Comment by frenzibyte** [comment]
~~All buttons should be configurable via keybinding configuration.~~ Hmm, I can't seem to get it working for me either, weirdly enough.

**Reply by nanashi-1** [reply]
https://user-images.githubusercontent.com/54083498/194781613-e9299ec4-217a-473d-a0c2-e8e2b5fa0e88.mp4

I configured my tablet buttons in [OpenTabletDriver](https://github.com/OpenTabletDriver/OpenTabletDriverl). Or you meant something else?

**Reply by frenzibyte** [reply]
Pressing a tablet auxiliary button should assign the action to that button itself (appearing as `TABLET AUX 1`, for first button), rather than to a mapped key in the `OpenTabletDriver` app.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu-framework/pull/5462
